launchable/commands/subset.py

```diff
@@ -141,6 +141,14 @@
     help="If you want to only send test reports, please use this option",
     is_flag=True,
 )
+@click.option(
+    '--lineage',
+    'lineage',
+    help='Set lineage name',
```

Comment:
[must]
`lineage` is unclear term. It is better to explain what is lineage and how it changes subset command.